clients/java/.settings/org.eclipse.buildship.core.prefs

```diff
@@ -0,0 +1,8 @@
+auto.sync=false
+build.scans.enabled=false
+connection.gradle.distribution=GRADLE_DISTRIBUTION(WRAPPER)
+connection.project.dir=
+eclipse.preferences.version=1
+gradle.user.home=
+offline.mode=false
+override.workspace.settings=true
```

Comment:
I think we shouldn't check this file in. When I opened the project in VS Code it modified the file. Seems like it might be a per-user thing...